editor/animation_track_editor.cpp

```diff
@@ -7007,25 +7016,31 @@ void AnimationTrackEditor::_selection_changed() {
 	}
 }
 
+void AnimationTrackEditor::_update_snap_unit() {
+	if (step->get_value() < 0) {
```

Comment:
```suggestion
	if (step->get_value() <= 0) {
```
Surely? Otherwise it'll still divide by zero below